conf/nginx.conf

```
@@ -20,12 +14,14 @@ location ^~ __PATH__/ {
   more_set_headers "X-Permitted-Cross-Domain-Policies: none";
   more_set_headers "Referrer-Policy: no-referrer";
 
-  #index index.php;
+  index index.php index.html index.htm;
```

```suggestion
  index index.php index.html index.htm;
  
    if (!-e $request_filename) {
    rewrite ^(.+)$ /index.php?q= last;
  }
```

The part:
```
  if (!-e $request_filename) {
    rewrite ^(.+)$ /index.php?q= last;
  }
```

is needed, otherwise logging in won't work and fail with a HTTP 405 error.